### ISSUE TITLE
Plugins are unreachable from JS

### ISSUE THREAD

**Issue by fmgasparino** [Telerik-Verified-Plugins/WKWebView#54]
When I use WKWebView, all the plugins working on the UIWebView are not reachable. Cordova exec does not execute. I'm using Cordova 3.5 for the project. Not sure if I missed any step but it seems straight forward by the command line.

Insights? 

Thank you and keep up with the good work. Performance improvements on the WKWebView is amazing! =)

**Comment by Scorpie** [comment]
I'm having the same issue and am also pointing to an external URL.

**Comment by fmgasparino** [comment]
@EddyVerbruggen, is there a limitation for local files only with WKWebView? Thank you!

**Comment by EddyVerbruggen** [comment]
@fmgasparino how are you loading the external file? Is it the `<content src="index.html" />` in `config.xml`, or something different? Please provide me some details. The exact host may provide some info as well because f.i. `<content src="http://www.telerik.com" />` works just fine in my test app.

**Comment by fmgasparino** [comment]
Yes, I'm loading the external URL from the config file. The URL loads fine, the problem is that canInitWithRequest is not called from external links, therefore plugins don't work...

I will try hosting an external app and will publish the link here.

Thanks @EddyVerbruggen
